### ISSUE TITLE
Create an in-depth documentation for the README.md file

### ISSUE THREAD

**Issue by kenny-io** [shardeum/archive-server#54]
Update the READM file in this repository to be more in-depth and cover the project in more detail.

A potential table of content for the file could look like:

* Overview of the project
* How to get started or local setup instruction
* Call out any specific configs or files that is a must-know for contributors
* How to contribute to the project, any special instructions

**Comment by rembertdesigns** [comment]
I would like to assist with this

**Comment by kenny-io** [comment]
> I would like to assist with this

@rembertdesigns go for it! 💪🏽

**Comment by EtimfonIkpong** [comment]
@kenny-io , can I take this up?

**Comment by kenny-io** [comment]
> @kenny-io , can I take this up?

@Etimfon007 Yes, go for it and let us know here if you need help with anything

**Comment by EtimfonIkpong** [comment]
Alright then. If I need any help I'll let you know @kenny-io
